libraries/matchers.rb

```diff
@@ -29,6 +29,7 @@
     oneview_fc_network:                 standard_actions,
     oneview_fcoe_network:               standard_actions,
     oneview_firmware:                   [:add, :remove, :create_custom_spp],
+    oneview_id_pool:                    [:enable, :disable, :allocate_list, :allocate_count, :collect_ids, :validate],
```

Comment:
A suggestion:

Perhaps instead of having many different actions for each thing, it actually makes sense to pack those together into an `update` action.
I'd imagine at least enable and disable could be inside an `update` action.

Also, `validate` does not seem to me like an action, it seems like something you'd call internally, perhaps before allocating a list. 

If this is similar to a get, it might be out of place. Does it cause any change on the appliance?